Testing with Docker fails on macOS Arm Processor

### Overview

If I follow [the README at "Testing" → "B. Via Docker"](https://github.com/DesignSafe-CI/DS-User-Guide/blob/ffc4547/README.md#a-via-python), **on macOS Arm processor**[^1], the build fails:

[^1]: I.e. a Mac with an M1 or M2 chip.

<details><summary><code>no match for platform in manifest: not found
</code></summary>

```
make build
docker-compose -f ./docker-compose.yml build
WARN[0000] /Users/wbomar/Code/DS-User-Guide/docker-compose.yml: `version` is obsolete 
[+] Building 1.3s (3/3) FINISHED                                                                                    docker:desktop-linux
 => [docs internal] load build definition from Dockerfile                                                                           0.0s
 => => transferring dockerfile: 635B                                                                                                0.0s
 => ERROR [docs internal] load metadata for docker.io/taccwma/tacc-docs:v0.6.0                                                      1.2s
 => [docs auth] taccwma/tacc-docs:pull token for registry-1.docker.io                                                               0.0s
------
 > [docs internal] load metadata for docker.io/taccwma/tacc-docs:v0.6.0:
------
failed to solve: taccwma/tacc-docs:v0.6.0: failed to resolve source metadata for docker.io/taccwma/tacc-docs:v0.6.0: no match for platform in manifest: not found
make: *** [build] Error 17
```

</details>

### Solutions

Wait for support from other software:
- https://github.com/TACC/TACC-Docs/issues/23#issuecomment-2195191138

### Workaround

1. Clone https://github.com/TACC/TACC-Docs.

2. Build & Start **TACC-Docs** via Docker.

3. Save the build as an image e.g.\
    `docker commit tup_docs`

4. Tag the image e.g.\
    `docker tag sha256:__SHA_OF_COMMIT__ local/tacc-docs:__ANY_IDENTIFIER__`

5. Edit https://github.com/TACC/DS-User-Guide `/Dockerfile`.

6. Change `FROM` line to use new local image e.g.
    - from `FROM taccwma/tacc-docs:v0.6.0`
    - to `FROM local/tacc-docs:__ANY_IDENTIFIER__`

7. Build & Start **DS-User-Guide** via Docker.

> [!IMPORTANT]
> Do **not** commit this change.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM taccwma/tacc-docs:v0.10.1
+FROM taccwma/tacc-docs:v0.12.0
 
 # To archive TACC content, before replacing it
 RUN mv /docs /docs-from-tacc

--- a/README.md
+++ b/README.md
@@ -65,9 +65,6 @@ How to Contribute **Other Changes**:
 
 ### B. Via Docker
 
-> [!WARNING]
-> This method does **not** work on macOS with an M1 nor M2 chip ([details](https://github.com/DesignSafe-CI/DS-User-Guide/issues/51)).
-
 0. Have Docker installed.\
     <sup>We recommend doing so via [Docker-Desktop](https://www.docker.com/products/docker-desktop).</sup>
 1. Navigate into your clone of this repository.

